Fix test_Sparse_to_Sparse_copy__cuda_bfloat16 failure

pearu opened the pull request:
Fixes #72892

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #73158
* __->__ #73157
* #73155

Differential Revision: [D34398986](https://our.internmc.facebook.com/intern/diff/D34398986)

pearu committed:
Fix test_Sparse_to_Sparse_copy__cuda_bfloat16 failure

[ghstack-poisoned]

cpuhrsch commented:
@cpuhrsch has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34398986).

pearu committed:
Update on "Fix test_Sparse_to_Sparse_copy__cuda_bfloat16 failure"

Fixes #72892


Differential Revision: [D34398986](https://our.internmc.facebook.com/intern/diff/D34398986)

[ghstack-poisoned]

pearu committed:
Update on "Fix test_Sparse_to_Sparse_copy__cuda_bfloat16 failure"

Fixes #72892


Differential Revision: [D34398986](https://our.internmc.facebook.com/intern/diff/D34398986)

[ghstack-poisoned]

pearu committed:
Update on "Fix test_Sparse_to_Sparse_copy__cuda_bfloat16 failure"

Fixes #72892

* #73155

[ghstack-poisoned]

cpuhrsch commented:
@cpuhrsch has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34512700).

cpuhrsch commented:
@cpuhrsch has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34398986).
